graphdatascience/tests/unit/test_call_params.py

```diff
@@ -14,3 +16,47 @@ def test_empty_params() -> None:
     params = CallParameters()
 
     assert params.placeholder_str() == ""
+
+
+def test_get_job_id() -> None:
+    # empty params
+    params = CallParameters({})
+    job_id = params.get_job_id()
+    assert job_id is None
+
+    # empty job id
+    params = CallParameters(config={job_id: None})
+    job_id = params.get_job_id()
+    assert job_id is None
+
+    # job_id given
+    params = CallParameters(config={"job_id": "foo"})
+    job_id = params.get_job_id()
+    assert job_id == "foo"
+
+    # jobId given
+    params = CallParameters(config={"jobId": "bar"})
+    job_id = params.get_job_id()
+    assert job_id == "bar"
```

Comment:
I wonder why we've got both of these

Reply:
also wondered why. I could see some usage of `job_id` inside the ArrowQueryRunner, but these are not passed inside a config param.
But for this PR I didnt want to touch as its more easy to introduce another bug if we just remove the case.

Reply:
yeah that's good thinking.